<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DePack</class>
 <widget class="QDialog" name="DePack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>解包工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextBrowser" name="TextBrowserExplain">
     <property name="placeholderText">
      <string>此用具用于拆解apk文件，查看AndroidManifest.xml和布局相关内容，smali文件代码不能反解到java，如需要反解smali文件的，请自行下载相应的工具去解。                                          注意：解包会清空解包存储目录的所有内容，所以选择解包存储位置时请慎重。</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="LabelOri">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Apk文件位置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineEditOri"/>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonScanOri">
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="LabelResult">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>解包存储位置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineEditResult"/>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonScanResult">
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="LabelProcess">
       <property name="text">
        <string>解包中</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="ProgressBarDePack">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOk">
       <property name="text">
        <string>解包</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
